lms/djangoapps/teams/static/teams/js/views/teams_tab.js

```diff
@@ -28,27 +57,148 @@
                                this.$el.text(this.text);
                            }
                        });
-                       this.tabbedView = new TabbedView({
-                           tabs: [{
-                               title: gettext('My Teams'),
-                               url: 'teams',
-                               view: new TempTabView({text: 'This is the new Teams tab.'})
-                           }, {
-                               title: gettext('Browse'),
-                               url: 'browse',
-                               view: new TopicsView({
-                                   collection: options.topicCollection
+                       this.topicsCollection = new TopicCollection(
+                           this.topics,
+                           {url: options.topics_url, course_id: this.course_id, parse: true}
+                       ).bootstrap();
+                       this.mainView = this.tabbedView = new ViewWithHeader({
+                           header: new HeaderView({
+                               model: new HeaderModel({
+                                   description: gettext("Course teams are organized into topics created by course instructors. Try to join others in an existing team before you decide to create a new team!"),
```

Comment:
Thanks, @explorerleslie 
How about choosing not to mention creating teams at all (so as not to draw attention to that ability) and leave it to course teams to tell students that they can do it?
"See all teams in your course, organized by topic. Join a team to collaborate with other learners who are interested in the same topic as you are."